src/constants/string_mappings.js

```diff
@@ -0,0 +1,37 @@
+/**
+* Copyright 2012-2016, Plotly, Inc.
+* All rights reserved.
+*
+* This source code is licensed under the MIT license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+
+
+'use strict';
+
+// N.B. HTML entities are listed without the leading '&' and trailing ';'
+
+module.exports = {
+
+    entityToUnicode: {
+        'mu': 'μ',
+        'amp': '&',
+        'lt': '<',
+        'gt': '>',
+        'quot': '\"',
```

Comment:
`&quot;` support makes me really uncomfortable since it's been part of so many of our XSS attacks (inserting a `"` sometimes allows an entity to be terminated early).

Is this definitely needed?

(The rest looks OK to me.)